### ISSUE TITLE
Bug on Photo application

### ISSUE THREAD

**Issue by gpbreier** [BenSova/Patched-Sur#418]
I've installed bigsur by the patchedsur and its work very well!
Otherside I work with photos and now, for me don't appear the library link in the sidebar as in other versions appear.

now is look as: 

![Screen Shot 2021-03-18 at 23 05 55](https://user-images.githubusercontent.com/80932879/111721528-d5212080-883e-11eb-9fde-c48bcd0c32e3.png)

before as it: 

![Screen Shot 2021-03-18 at 23 05 55](https://user-images.githubusercontent.com/80932879/111721613-0c8fcd00-883f-11eb-80e1-cc3912a94bb6.png)

I cant see the library link, on top left and select the day month and year as the old version

At contact with apple they don't find a solution and ask about my version mac and software.
The library is there, but de sidebar menu dont

Someone can help me to restore this important link?

**Comment by ozziepadilla** [comment]
> I've installed bigsur by the patchedsur and its work very well!
> Otherside I work with photos and now, for me don't appear the library link in the sidebar as in other versions appear.
> 
> now is look as:
> 
> ![Screen Shot 2021-03-18 at 22 58 32](https://user-images.githubusercontent.com/80932879/111722117-f5051400-883f-11eb-989d-5d70cdeec5ee.png)
> 
> before as it:
> 
> ![Screen Shot 2021-03-18 at 23 05 55](https://user-images.githubusercontent.com/80932879/111721528-d5212080-883e-11eb-9fde-c48bcd0c32e3.png)
> I cant see the library link, on top left and select the day month and year as the old version
> 
> At contact with apple they don't find a solution and ask about my version mac and software.
> The library is there, but de sidebar menu dont
> 
> Someone can help me to restore this important link?

First you shouldn’t contact Apple since our Mac 💻 and iMac 🖥 doesn’t support macOS Big Sur they won’t help us , solution , log out of your Apple ID iCloud account restart your Mac then log in again launch photos app , don’t close it out leave in background, depends of the amount of pictures and albums you have it will takes time to download everything, check system preferences go toy your Apple ID and verify if all settings are ON and also find my Mac must be in and every settings make sure the toggle is on , patience is the key for this , again don’t contact Apple for support because they won’t help you

**Comment by ozziepadilla** [comment]
By the way isn’t a BUG is just configuration that needs to be refreshed log out and log in

**Comment by ozziepadilla** [comment]
Hi, if still the pictures library didn’t work, go to the photos menu and check if iCloud photos is on and all toggles are on if not turn turn them on

**Comment by ozziepadilla** [comment]
Check this out 
![Uploading 9F1E3357-9D71-435B-80D0-BBEB9652021E.jpeg…]()

**Comment by gpbreier** [comment]
Hi Ozziepadilla!

I logout of my Apple ID, reset my macbook, check if every thing is ok and relauch the apple photos and doesn't work

I check the library in settings and when a click on the library, its work corrects.

I cant see the image that you send, but I image that is a setting of library in preferences. is it correct?

So I think that this is a bug and not a just setting.

**Comment by ozziepadilla** [comment]
> Hi Ozziepadilla!
> 
> I logout of my Apple ID, reset my macbook, check if every thing is ok and relauch the apple photos and doesn't work
> 
> I check the library in settings and when a click on the library, its work corrects.
> 
> I cant see the image that you send, but I image that is a setting of library in preferences. is it correct?
> 
> So I think that this is a bug and not a just setting.

Ok do this , go to www.iCloud.com log in with your Apple ID and go to the photos app , let me know if you can see all your pictures

**Comment by ozziepadilla** [comment]
> Hi Ozziepadilla!
> 
> I logout of my Apple ID, reset my macbook, check if every thing is ok and relauch the apple photos and doesn't work
> 
> I check the library in settings and when a click on the library, its work corrects.
> 
> I cant see the image that you send, but I image that is a setting of library in preferences. is it correct?
> 
> So I think that this is a bug and not a just setting.

Couldn’t be a bug because nobody else has report it something like that because everything works normally, we did an update not a clean installation

**Comment by ozziepadilla** [comment]
> Hi Ozziepadilla!
> 
> I logout of my Apple ID, reset my macbook, check if every thing is ok and relauch the apple photos and doesn't work
> 
> I check the library in settings and when a click on the library, its work corrects.
> 
> I cant see the image that you send, but I image that is a setting of library in preferences. is it correct?
> 
> So I think that this is a bug and not a just setting.

If you are using the same Apple ID in your iPhone 📲 check your iPhone in settings and turn off/on iCloud photos and everything must be on too

**Comment by BenSova** [comment]
If you open System Preferences then select your iCloud account, you should see a list of options. One of them is Photos, make sure there's a check there. If it's not that, I'm not sure why Big Sur is acting up, but what Mac is this on?

**Comment by ozziepadilla** [comment]
> If you open System Preferences then select your iCloud account, you should see a list of options. One of them is Photos, make sure there's a check there. If it's not that, I'm not sure why Big Sur is acting up, but what Mac is this on?

Hi Ben , yeah that’s weird , when I did my first updates to all versions of Big Sur I had to reload my printer all the time

**Comment by ozziepadilla** [comment]
> If you open System Preferences then select your iCloud account, you should see a list of options. One of them is Photos, make sure there's a check there. If it's not that, I'm not sure why Big Sur is acting up, but what Mac is this on?

what I think is icloud syncing its out I have check that on system status in iCloud.com

**Comment by ozziepadilla** [comment]
> Hi Ozziepadilla!
> 
> I logout of my Apple ID, reset my macbook, check if every thing is ok and relauch the apple photos and doesn't work
> 
> I check the library in settings and when a click on the library, its work corrects.
> 
> So I think that this is a bug and not a just setting.

Make sure your Mac system preferences photos is enable
![FDFD6A00-1CDF-473F-BDFF-125F378A5A37](https://user-images.githubusercontent.com/78187370/111795361-7c34a500-889d-11eb-8ea6-b49e168a1ebb.png)
![58425C04-2BDB-45CB-BAF8-E645B8CE8806](https://user-images.githubusercontent.com/78187370/111795369-7d65d200-889d-11eb-9d84-ab4be2ed3926.png)

**Comment by gpbreier** [comment]
Hi ozziepadilla, I've read what that you wrote and started a new way as your suggest.

I clean up my MacBook Pro 2011 (initial) and install Highsierra macOS first. I check al settings that you recommend and its ok

after that, I download the patched of dos dude of macOS Catalina, I've just updated (not a clean version) without install the previous Mojave macOS, and now the menu library doesn't appear more.

![Screen Shot 2021-03-19 at 22 47 46](https://user-images.githubusercontent.com/80932879/111855850-b0dc4700-8905-11eb-96dc-f904c9e1c0bf.png)

All the photos are there, but 

the correct is 
Photos (library don't clicked...) 
  Library  (click menu)
  Memory (click menu)
  ...

and this is the bug that I thought as a configuration and because this I used the Apple support, but they don't imaged that my Mac was supported 

Tomorrow I'll start again and try install Mojave before the Catalina to see what's happen..

All the library is ok and work, Mac and iCloud. All photos is OK

other bugs that I observed is that the maps appear a just black image with a blue point focus me, and inside the maps similar, a black square.

thanks for you support

**Comment by gpbreier** [comment]
the bug is why the menu library doesnt appear below the title Photos

**Comment by ozziepadilla** [comment]
> the bug is why the menu library doesnt appear below the title Photos

Ok now I understand, you use Dosdude Catalina patchet and your Mac had macOS high Sierra installed and not macOS Mojave then you upgrade to Big Sur  , you get me confuse honestly I don’t know dosdude Catalina how works or if it’s compatible for those Mac 💻 that doesn’t support Mojave

**Comment by ozziepadilla** [comment]
> the bug is why the menu library doesnt appear below the title Photos

What was the last macOS that was compatible with your MacBook 💻?

**Comment by BenSova** [comment]
I believe it's High Sierra

**Comment by ozziepadilla** [comment]
> I believe it's High Sierra

Yup me too , I don’t know the compatibility of dosdude Catalina patchet if supports to upgrade from high Sierra without Mojave

**Comment by ozziepadilla** [comment]
> I believe it's High Sierra

ben did you have installed macOS Big Sur 11.3 beta 4 ? If yes how works ?

**Comment by BenSova** [comment]
> Yup me too , I don’t know the compatibility of dosdude Catalina patchet if supports to upgrade from high Sierra without Mojave

It does.

> ben did you have installed macOS Big Sur 11.3 beta 4 ? If yes how works ?

Beta 4 was taken down from my catalog a while ago because it didn't play nice with the macOS updater. I've also stayed on the macOS releases recently, mostly because I'm not in the mood for updating macOS all the time.

**Comment by ozziepadilla** [comment]
> > Yup me too , I don’t know the compatibility of dosdude Catalina patchet if supports to upgrade from high Sierra without Mojave
> 
> It does.
> 
> > ben did you have installed macOS Big Sur 11.3 beta 4 ? If yes how works ?
> 
> Beta 4 was taken down from my catalog a while ago because it didn't play nice with the macOS updater. I've also stayed on the macOS releases recently, mostly because I'm not in the mood for updating macOS all the time.

Well next week Apple will release iOS 14.5 RC and iPadOS 14.5, watchOS too together  with new devices also I read Big Sur 11.3 too

**Comment by gpbreier** [comment]
Hi Guys! 
To refresh the instructions.
I understood that to this it works I need to just update and not a clean installation.

After that I did a clean installation of High Sierra macOS because this is the last OS supported by my MacBook Pro 2011 (initial).
I sign in my iCloud account and everything is OK e perfect. All the photos are there.

After that I would like install the BigSur, but as I read, a need to install macOS Catalina before to install macOs BigSur. 

Following the instructions on BenSova page, he recommends use the patched of dosDude to install macOS Catalina.

So at that moment, to be fast I though that was not necessary update first to macOS Mojave, because the timeline is HighSierra, Mojave, Catalina, BigSur.

I checked all the settings and the configurations in HighSierra, and these is all works...

So I stared to update to macOS Catalina by dos dude. I was imagined that everything is will ok... so When I opened the photos the menu option library, doesn't stay there more. 

So I stoped the process because if in Cantalina doesn't work, in the next step it will not work too.

**Comment by gpbreier** [comment]
My friend take a screenshot of his MacBook Air support for bigger

this is how appear without the bug

![Screen Shot 2021-03-20 at 11 20 54](https://user-images.githubusercontent.com/80932879/111874496-51695000-8974-11eb-9416-c76ee11aad9b.png)

And this is how appear with the bug
![111722117-f5051400-883f-11eb-989d-5d70cdeec5ee](https://user-images.githubusercontent.com/80932879/111874529-8fff0a80-8974-11eb-81c4-96a93911c01b.png)

**Comment by gpbreier** [comment]
the menu Library is out of the program.

**Comment by gpbreier** [comment]
And I try before a clean installation of big sur, and now I tried a update of system and stop in the step Catalina because I observed the problem

**Comment by ozziepadilla** [comment]
> And I try before a clean installation of big sur, and now I tried a update of system and stop in the step Catalina because I observed the problem

Clean install of Big Sur isn available, did you install Catalina and let the photos library to downloading before run patched Sur ?

**Comment by ozziepadilla** [comment]
In  other screenshots I can something, Big Sur “ Launcpad” looks the same , question, those screenshots are from your MacBook? Or someone else MacBook?

**Comment by ozziepadilla** [comment]
Ok do this first , check the Catalina patchet if supports your Mac 💻 with high Sierra if is supported then run Catalina patchet let it finish then go to check your photos library settings in system preferences Apple ID self taught iCloud  and select photos if isn’t selected and all options that aren’t select it then go to photos preferences if iCloud photos is enable si disable it then enable again , leave photos app in background don’t rush to run patchet Sur yet leave that all configurations finishing then after all that run patchet Sur , suggestion if you are using an iPhone or iPad or iPod make a iCloud back up first usually iCloud photos makes backups separately so do it just to have a copy of your actual library pictures

**Comment by ozziepadilla** [comment]
> And I try before a clean installation of big sur, and now I tried a update of system and stop in the step Catalina because I observed the problem

Do you hsvdd es the last Catalina patchet? If not download it from here and read the compatibility MacBook list
http://dosdude1.com/catalina/

**Comment by gpbreier** [comment]
> > And I try before a clean installation of big sur, and now I tried a update of system and stop in the step Catalina because I observed the problem
> 
> Clean install of Big Sur isn available, 

Yes, it is possible, I created I USB boot, format the HD and install after bigger

> did you install Catalina and let the photos library to downloading before run patched Sur ?
 Ive update the highSierra, download all the photos, let the photos library ok to start before the downloading the patched Catalina

**Comment by gpbreier** [comment]
![Screen Shot 2021-03-20 at 11 20 54](https://user-images.githubusercontent.com/80932879/111876721-cfcaef80-897e-11eb-8906-413cf63dee03.png)

This screen is from my friend with a MacBook Air supported with bigSur

**Comment by gpbreier** [comment]
![111874529-8fff0a80-8974-11eb-81c4-96a93911c01b](https://user-images.githubusercontent.com/80932879/111876924-9e065880-897f-11eb-8132-a10cb3ecdf59.png)

and this is from my Mac with big sur installed in a clean install

**Comment by gpbreier** [comment]
![Screen Shot 2021-03-20 at 13 27 54](https://user-images.githubusercontent.com/80932879/111877077-50d6b680-8980-11eb-8ead-62e95bf8854f.png)

This is the screen with photos on Catalina after I update high Sierra, I can't work without the menu all photos. The library is ok, inside the Mac and in iCloud too

**Comment by ozziepadilla** [comment]
> > > And I try before a clean installation of big sur, and now I tried a update of system and stop in the step Catalina because I observed the problem
> > 
> > 
> > Clean install of Big Sur isn available,
> 
> Yes, it is possible, I created I USB boot, format the HD and install after bigger
> 
> > did you install Catalina and let the photos library to downloading before run patched Sur ?
> > Ive update the highSierra, download all the photos, let the photos library ok to start before the downloading the patched Catalina

Sorry but isn’t available, and you shouldn’t firmar your Macintosh HD because you are doing an upgrade from a old macOS to a new one

**Comment by BenSova** [comment]
He managed to get Patched Kexts, so he's fine there.

**Comment by BenSova** [comment]
I think the actual bug is that you don't have Metal Graphics Acceleration. I don't have much support for those Macs because of how limited macOS can get without Metal. I have Metal, and everything that you say is missing is there on mine. There's no way I can fix this bug.

**Comment by gpbreier** [comment]
Hi BenSova! Maybe this is the problem! Ok! for me, maybe its (the absence 0f menu Library) don't have connection with the metal graphics acceleration, but to the window of maps and window of places in APP Photos, I'm total agree.
